### ISSUE TITLE
FR: Open "previous" document in folder for reader+filemanager

### ISSUE THREAD

**Issue by shuvashish76** [koreader/koreader#12689]
**Does your feature request involve difficulty completing a task? Please describe.**
I've setup gesture for "On book" to perform following action to easily browse documents with just one-hand
suppose:
1. One-finger swipe > Right edge down > Reader > Open next document in folder

I want to have an action so that I can `Open previous document in folder` with `One-finger swipe > Right edge up` but the problem is there is no "Open previous document in folder" action. I can set "One-finger swipe > Right edge up > General > **Open previous document**" but it's not same as "Open previous document in folder"
Open previous document : this action changes 2 recent document back-and-forth only

**Describe the solution you'd like**
Add following actions for "On book"/Reader
- [x] Open next document in folder (already exist)
- [ ] Open previous document in folder

**Comment by KopiasCsaba** [comment]
Yes, yes it would be useful when you are browsing a lot of downloaded articles.
